app/src/main/res/menu/menu_mega_status.xml

```diff
@@ -1,9 +1,12 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     tools:context="com.eveningoutpost.dexdrip.MegaStatus">
-    <item
-        android:id="@+id/action_settings"
```

Comment:
Does the burger menu to access the settings still work if you remove these lines here?